pkg/controller/cassandracluster/reconcile_test.go

```diff
@@ -85,7 +85,9 @@ func TestFlipCassandraClusterUpdateSeedListStatusScaleDC2(t *testing.T) {
 	UpdateCassandraClusterStatusPhase(cc, status)
 
 	//Flip with AutoUpdateSeedList= true -> update status
-	FlipCassandraClusterUpdateSeedListStatus(cc, status)
+	if err := FlipCassandraClusterUpdateSeedListStatus(cc, status); err != nil {
+		assert.Fail(err.Error())
+	}
```

Comment:
All those checks are wrong IMO. Here it seems we don't know if it will happen or not, meaning the test is not deterministic which is not good. It's a unit test, it must be called or not but we need to know it.

Reply:
Hum ok. So do think that '_' instead of 'err' it's better since this test is not made to trigger this issue ?

Reply:
Nah. What I'm saying is that a test should never have tests like this. The conditions are given and the function must error or not. In that test it seems to depend which doesn't make any sense and should be removed/fixed